alws/dramatiq/products.py

```diff
@@ -188,8 +193,12 @@ async def set_platform_for_build_repos(
     # name of a build's repo:
     # some-platform-x86_64-some-build-id-debug-br
     repos_per_platform = {
-        f"{task.platform.name}-{repo.arch}-{build.id}-"
-        f"{repo_debug_dict[repo.debug]}": task.platform
+        # fmt: off
+        (
+            f"{task.platform.name}-{repo.arch}-{build.id}"
+            f"-{repo_debug_dict[repo.debug]}"
+        ): task.platform
+        # fmt: on
```

Comment:
how is `black` trying to correct that? as for me, it looks a bit weird to leave such comments as `fmt` in the code